tests/test_cli_change_lb.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from control.server import GatewayServer
+from control.cli import main as cli
+import grpc
+from control.proto import gateway_pb2 as pb2
+from control.proto import gateway_pb2_grpc as pb2_grpc
+import copy
+
+image = "mytestdevimage"
+image2 = "mytestdevimage2"
+pool = "rbd"
+subsystem = "nqn.2016-06.io.spdk:cnode1"
+anagrpid = "1"
+anagrpid2 = "2"
+config = "ceph-nvmeof.conf"
+
+@pytest.fixture(scope="module")
+def two_gateways(config):
+    """Sets up and tears down two Gateways"""
+    nameA = "GatewayAA"
+    nameB = "GatewayBB"
+    sockA = "spdk_GatewayAA.sock"
+    sockB = "spdk_GatewayBB.sock"
+    config.config["gateway-logs"]["log_level"] = "debug"
+    addr = config.get("gateway", "addr")
+    configA = copy.deepcopy(config)
+    configB = copy.deepcopy(config)
+    configA.config["gateway"]["name"] = nameA
+    configA.config["gateway"]["override_hostname"] = nameA
+    configA.config["spdk"]["rpc_socket_name"] = sockA
+    portA = configA.getint("gateway", "port") + 1
+    configA.config["gateway"]["port"] = str(portA)
+    discPortA = configA.getint("discovery", "port") + 1
+    configA.config["discovery"]["port"] = str(discPortA)
+    configB.config["gateway"]["name"] = nameB
+    configB.config["gateway"]["override_hostname"] = nameB
+    configB.config["spdk"]["rpc_socket_name"] = sockB
+    portB = portA + 2
+    discPortB = discPortA + 1
+    configB.config["gateway"]["port"] = str(portB)
+    discPort = configB.getint("discovery", "port") + 1
+    configB.config["discovery"]["port"] = str(discPortB)
+    configB.config["spdk"]["tgt_cmd_extra_args"] = "-m 0x02"
+
+    with (GatewayServer(configA) as gatewayA, GatewayServer(configB) as gatewayB):
+        gatewayA.set_group_id(0)
+        gatewayB.set_group_id(1)
+        gatewayA.serve()
+        gatewayA.ceph_utils.execute_ceph_monitor_command("{" + f'"prefix":"nvme-gw create", "id": "{nameA}", "pool": "{pool}", "group": ""' + "}")
```

Comment:
since 'nvme-gw create' is sent to the monitor server, lines 46-47 (calling set_group_id) are not required.

Reply:
Will remove it.